src/main/java/ca/corefacility/bioinformatics/irida/model/user/User.java

```diff
@@ -350,6 +358,10 @@ public void setSystemRole(Role systemRole) {
 		this.systemRole = systemRole;
 	}
 
+	public void setUserType(UserType userType) {
+		this.userType = userType;
```

Comment:
I made the comment below, but instead of allow null here and catching it with the null constraint, why not just raise an exception here as the code should never allow you to specify null. That way we don't need the translation that should never be shown to the user.